.github/workflows/docs.yml

```
@@ -35,6 +35,7 @@ jobs:
         cmake -DCMAKE_INSTALL_PREFIX:PATH=/usr -S alure -B alure/build
         sudo cmake --build alure/build --parallel `nproc` --target install
         rm -fr alure
+        python -m pip install sphinx_rtd_theme
         python -m pip install Sphinx .
```

LGTM, except for one minor spot:

```suggestion
        python -m pip install Sphinx sphinx_rtd_theme .
```

Imma apply this myself and merge.  Thanks for your patience!

---

yay! Thanks for your patience as well!

---

They changed the guidelines for approving hacktoberfest PRs,  to avoid spammy PRs. 

I hope my changes are sufficient and you approve them. Thanks!

You may use this as a guide: https://hacktoberfest.digitalocean.com/hacktoberfest-update

---

I think this one should count already:

> **Submitted in a repo with the hacktoberfest topic** AND    
> **during the month of October** AND (    
>   **The PR is merged** OR    
>   The PR is labelled as hacktoberfest-accepted by a maintainer OR    
>   The PR has been approved)